accession-commons-test/src/main/java/uk/ac/ebi/ampt2d/test/testers/AccessioningServiceTester.java

```diff
@@ -94,7 +95,19 @@ public AccessionWrapperCollectionTester get(List<TestModel> models) {
     public AccessionWrapperCollectionTester getAccessions(String... accessionIds) {
         return addToCollection(singleVersionResults,
                 new AccessionWrapperCollectionTester(() ->
-                        accessioningService.getByAccessions(Arrays.asList(accessionIds))));
+                        getLatestVersionOfAccessions(accessionIds)));
+    }
+
+    private List<AccessionWrapper<TestModel, String, String>> getLatestVersionOfAccessions(String[] accessionIds) {
+        List<AccessionWrapper<TestModel, String, String>> accessionWrappers = new ArrayList<>();
+        try {
+            for (String accessionId : accessionIds) {
+                accessionWrappers.add(accessioningService.getByAccession(accessionId));
+            }
+        } catch (Exception e) {
```

Comment:
Do not hide exceptions by printing the stack trace, even more so in test-related classes.

Reply:
+1